### PR TITLE
Enhance crypto policies checks to handle packages not installed

### PR DESCRIPTION
**Pull request by yuumasato**
#### Description:
- Definition `configure_bind_crypto_policy` will pass if Bind is not installed.
- Definition `configure_libreswan_crypto_policy` will pass if Libreswan is not installed.

#### Rationale:

- If Bind is not installed, there is no need to check whether it is overriding system-vide crypto policy configuration
- If Libreswan is not installed, there is no need to check whether it is overriding system-vide crypto policy configuration

### CODE REVIEW

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml [hunk 1]
```diff
@@ -6,9 +6,10 @@
         <platform>multi_platform_fedora</platform>
         <platform>Red Hat Enterprise Linux 8</platform>
       </affected>
-      <description>BIND should be configured to use the system-wide crypto policy setting.</description>
+      <description>BIND is not installed or is configured to use the system-wide crypto policy setting.</description>
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml]: This doesn't go here. This goes into OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml [hunk 2]
```diff
@@ -9,6 +9,8 @@ description: |-
     BIND is supported by crypto policy, but the BIND configuration may be
     set up to ignore it.
 
+    Either BIND is not installed, or it is configured to use the system-wide defined crypto policy.
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml]: This doesn't go here. This goes into OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/oval/shared.xml [hunk 3]
```diff
@@ -6,9 +6,10 @@
         <platform>multi_platform_fedora</platform>
         <platform>Red Hat Enterprise Linux 8</platform>
       </affected>
-      <description>Libreswan should be configured to use the system-wide crypto policy setting.</description>
+      <description>Libreswan is not installed or is configured to use the system-wide crypto policy setting.</description>
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/oval/shared.xml]: This doesn't go here. This goes into OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml [hunk 4]
```diff
@@ -9,6 +9,8 @@ description: |-
     Libreswan is supported by system crypto policy, but the Libreswan configuration may be
     set up to ignore it.
 
+    Either Libreswan is not installed, or it is configured to use the system-wide defined crypto policy.
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml]: This doesn't go here. This goes into OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml (hunk 1 above)
Comment (redhatrises) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml]: And really... this change doesn't need to take place in the descriptions.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml (hunk 2 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml]: Moved to OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml (hunk 4 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml]: Moved to OCIL.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml]: Are you strongly against having the check description like this?

Comment (redhatrises) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml]: Yes.... we don't do this for other checks. And really, this is an OCIL clause statement.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml [hunk 5]
```diff
@@ -26,6 +26,10 @@ ocil_clause: |-
     <pre>include "/etc/crypto-policies/back-ends/bind.config";</pre> directive
 
 ocil: |-
+    Either BIND is not installed, or it is configured to use the system-wide defined crypto policy.
+
+    {{{ ocil_package(package="bind") }}}
+
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml]: This should be in the OCIL clause and `{{{ ocil_package(package="bind") }}}` is not needed.

File: linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml [hunk 6]
```diff
@@ -26,6 +26,10 @@ ocil_clause: |-
     <tt>/etc/ipsec.conf</tt> does not contain <tt>include /etc/crypto-policies/back-ends/libreswan.config</tt>
 
 ocil: |-
+    Either Libreswan is not installed, or it is configured to use the system-wide defined crypto policy.
+
+    {{{ ocil_package(package="libreswan") }}}
+
```
Comment (redhatrises) [inline linux_os/guide/system/software/integrity/crypto/configure_libreswan_crypto_policy/rule.yml]: This should be in the OCIL clause and `{{{ ocil_package(package="libreswan") }}}` is not needed.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml (hunk 1 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/oval/shared.xml]: Reverted. The check for package bind installed is acting as an applicability check, not as essential part of the check.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml (hunk 5 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml]: Edited and moved to OCIL clause. Hope it makes sense.

File: linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml (hunk 2 above)
Comment (yuumasato) [reply linux_os/guide/system/software/integrity/crypto/configure_bind_crypto_policy/rule.yml]: Relocated to OCIL clause.

**Comment by redhatrises** [comment]
Perfect! Great job @yuumasato!

**Pull request merged**